### DISCUSSION TITLE
Commercial use of TTS v1 & Open Source

### DISCUSSION THREAD

**Discussion by kainscu** [coqui-ai/TTS#2960] [General]
Hello, since Coqui is labelled as open source and being marketed as such, I realised even though it's open source I can not use it commercially without paying.

I have seen that 2 days ago the XTTS Model has been licensed under the community license, which does not allow you to use it commercially. So I wanted to ask for some exact clarification before using coqui commercially.

1. When I deploy coqui on my own systems and use it with my own server, do I need to pay a license fee, or pay per hour?
2. Is the older TTS1 (not xtts) also not available commercially or are those models available under open source?
3. If I train my own model (i am aware this is super expensive), can I use coqui commercially without paying coqui company?

I saw that on many boards the people are unhappy with the route coqui has decided to go (marketing as open source, but not allowing commercial use without payment) but I understand the choice for this route since otherwise it would be hard monetarily. On the other hand, for small indie teams the costs of 5-10$ per hour is too high, therefore I need to get it running for cheaper (or switch to azure voice ai, since they charge ~1$ per hour)

**Comment by erogol** [comment]
The Coqui code is released under the MPL license, allowing commercial use. However, it's important to note that the licenses for individual models may vary depending on the provider. Therefore, it is crucial to review the specific model license before using it.

Creating and maintaining training models involves significant costs, which necessitates financial support. While it would be ideal to release models for free, the reality is that resources are required to cover these expenses.

**Reply by imrnh** [reply]
Hey Mr. @erogol. As the company shutting down and further training won't be continued, can we use the trained models commercially without paying?

**Comment by knightfoxapps** [comment]
Yea what is the implication of the company shutting down? I understand that the model training wont continue but what does that mean for the code here?

**Comment by Tarik-123515** [comment]
hi
